projects/samples/howto/worlds/inverted_pendulum.wbt

```diff
@@ -17,11 +17,12 @@ WorldInfo {
 Viewpoint {
   orientation 0.05317585096836776 0.9887920240003923 0.1395086454202659 3.609723118221519
   position -6.644331534525101 4.261264054273784 -9.7759672742021
+  follow "robot:solid"
 }
 TexturedBackground {
 }
 Floor {
-  size 2 60
+  size 2 10000
```

Comment:
Really that big?

Reply:
Yes, when the solid speed increase, it can easily overtakes 1000 meters (due to the dimension of the robot). I would put an infinite plane here if I could.